plugins/destination/test/go.mod

```diff
@@ -4,10 +4,15 @@ go 1.21.4
 
 require (
 	github.com/apache/arrow/go/v15 v15.0.0-20240114144300-7e703aae55c1
+	github.com/cloudquery/codegen v0.3.12
 	github.com/cloudquery/plugin-sdk/v4 v4.28.0
+	github.com/invopop/jsonschema v0.12.0
 	github.com/rs/zerolog v1.31.0
 )
 
+// github.com/cloudquery/jsonschema @ cqmain
```

Comment:
better move the replace to the end of the mod file (otherwise it might produce severla additiona sections & not keep the direct & indirect imports separated